website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,49 +1,122 @@
     If set, determines the vertical alignment for all cell (`td`) content in a table. Does not apply to table headers (`th`).
   </C.Property>
   <C.Property @name="caption" @type="string">
-    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text.
+    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text. Note: it is best practice to provide a table caption for users with assistive technology.
   </C.Property>
   <C.Property @name="...attributes">
     Supported for the `Hds::Table` component.
   </C.Property>
+  <C.Property @name="onSort" @type="function">
+    Callback function that is invoked when one of the sortable table headers is clicked (or has a keyboard interaction performed). The function receives the values of `sortBy` and `sortOrder` as arguments.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Tr
+
+The `Hds::Table::Tr` component is a template-only component.
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of a `th` or `td` element.
+
+It can contain `Hds::Table::Th` or `Hds::Table::Td` components.
+
+<Doc::ComponentApi as |C|>
+    <C.Property @name="...attributes">
+    This component supports the use of `...attributes`.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Th
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of a `th` or `td` element.
+
+If a `th` is passed as the first "cell" of a table body row, it has `scope="row"` automatically applied to it for accessibility purposes.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+    If set, determines the text alignment.
+  </C.Property>
+  <C.Property @name="width" @type="string" @valueNote="Any valid CSS" @default="left">
+    If set, determines the column width.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports the use of `...attributes`.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::ThSort
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of a `th` or `td` element.
+
+This is the component that supports the column sorting, and should be used instead of `Hds::Table::Th` if creating a custom implementation.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="onClick" @type="function">
```

Comment:
We tend to have `onSomething` events after the other normal arguments (with `...attributes` last)

Reply:
Changed it! 👍